internal/commands/app/create_test.go

```diff
@@ -379,8 +379,9 @@ Check out your app: cd ./test-app && realm-cli app describe
 
 		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: realmClient, Atlas: atlasClient}))
 
-		_, err = local.LoadApp(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name))
-		assert.Nil(t, err)
+		appPath := filepath.Join(profile.WorkingDirectory, cmd.inputs.Name)
+		_, err = local.LoadApp(appPath)
+		assert.Equal(t, err, local.ErrFailedToFindApp(appPath))
```

Comment:
rather than have to export the error, i'd just assert equal to `errors.New(...)` (with the expected string)